### DISCUSSION TITLE
Julia 1.7.3.pdf error 404

### DISCUSSION THREAD

**Discussion by shayandavoodii** [JuliaLang/julia#45418] [General]
It seems there are issues with downloading the `julia-1.7.3.pdf` file. I commented on this here:
https://github.com/JuliaLang/julia/issues/43317#issuecomment-1133845959. Does anyone else experiencing it?

**Comment by fredrikekre** [comment]
1.7.3 isn't released yet but fixed in https://github.com/JuliaLang/docs.julialang.org/commit/607db32809c9e1c3c7feaaef3026e697c77b7b44.

**Reply by shayandavoodii** [reply]
Thanks, But still, the version shown in [doc](https://docs.julialang.org/) is `1.7.3`. It would be best to change it to `1.7.2` since `1.7.3` isn't out yet.
